src/components/pages/module-view/ModuleViewContainer.jsx

```diff
@@ -2,69 +2,40 @@ import React, { useState } from 'react';
 import { Card } from 'antd';
 import styled from 'styled-components';
 
+//Component Styles
 const ModuleCard = styled(Card)`
   margin-bottom: 4%;
 `;
 
-//Test Data
-const moduleData = [
-  {
-    moduleName: 'Python Basics',
-    moduleTopics: [
-      {
-        moduleTopicName: 'Warmup',
-        moduleTopicContent: 'Test Description 12345',
-      },
-      {
-        moduleTopicName: 'Use Print Statement',
-        moduleTopicContent: 'Test Description 12345',
-      },
-      {
-        moduleTopicName: 'List Operations',
-        moduleTopicContent: 'Test Description 12345',
-      },
-    ],
-  },
-  {
-    moduleName: 'Linked Lists',
-    moduleTopics: [
-      {
-        moduleTopicName: 'Warmup',
-        moduleTopicContent: 'Test Description 12345',
-      },
-      {
-        moduleTopicName: 'Use Print Statement',
-        moduleTopicContent: 'Test Description 12345',
-      },
-      {
-        moduleTopicName: 'List Operations',
-        moduleTopicContent: 'Test Description 12345',
-      },
-    ],
-  },
-];
+const Description = styled.div`
+  margin-bottom: 2%;
+`;
 
 const ModuleViewContainer = props => {
```

Comment:
this component is great, just change the `key` attribute for `<ModuleCard />` and you'll be set.

Reply:
Totally on it, more soon with all the fixes